File: packages/mesh-io/typescript/src/extension-to-mesh-io.ts
const extensionToMeshIo = new Map([
  ['vtk', 'vtk'],
  ['byu', 'byu'],
  ['fsa', 'free-surfer-ascii'],
  ['fsb', 'free-surfer-binary'],
  ['mz3', 'mz3'],
  ['obj', 'obj'],
  ['off', 'off'],
  ['stl', 'stl'],
  ['swc', 'swc'],
  ['iwm', 'wasm'],
  ['iwm.cbor', 'wasm'],
  ['iwm.cbor.zst', 'wasm-zstd']
])

export default extensionToMeshIo

File: packages/mesh-io/python/itkwasm-mesh-io-emscripten/itkwasm_mesh_io_emscripten/extension_to_mesh_io.py
from collections import OrderedDict

extension_to_mesh_io = OrderedDict([
    ('.vtk', 'vtkPolyData'),
    ('.byu', 'byu'),
    ('.fsa', 'freeSurferAscii'),
    ('.fsb', 'freeSurferBinary'),
    ('.mz3', 'mz3'),
    ('.obj', 'obj'),
    ('.off', 'off'),
    ('.stl', 'stl'),
    ('.swc', 'swc'),
    ('.iwm', 'wasm'),
    ('.iwm.cbor', 'wasm'),
    ('.iwm.cbor.zst', 'wasmZstd'),
])
